jenkins/prepare-ps-build-docker.yml

```diff
@@ -35,7 +35,6 @@
                 label 'docker-32gb'
             }
             options {
-                compressBuildLog()
```

Comment:
why not?

Reply:
This came from sync with jenkins, will check & re-add it back

Reply:
Re-added in 9fb97b4